app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/SubscriptionDownloadController.scala

```diff
@@ -42,7 +42,7 @@ class EoriDownloadController @Inject() (
     implicit request => _: LoggedInUserWithEnrolments =>
       cdsFrontendDataCache.sub02Outcome.map {
         case Sub02Outcome(processedDate, fullName, Some(eori)) =>
-          Right(eoriNumberDownloadView(eori, fullName, processedDate).body)
+          Right(subscriptionDownloadView(eori, fullName, processedDate).body)
```

Comment:
We can have a common controller for both standalone and combi journeys

Reply:
I have removed the separate controller and now using the common controller for both standalone and combi journeys.